Makefile

```diff
@@ -15,8 +15,8 @@ ODF_VERSION ?= 4.10
 
 
 
-ifeq ($(BRANCH),)
-BRANCH := $(RELEASE)
+ifneq ($(TAG),)
```

Comment:
This should go before the `FULL_PIPE_IMAGE_TAG` and `FULL_UI_IMAGE_TAG` definitions as it affects the image tag name when building the image

Reply:
correct - well sometime we use the NAME and BRANCH and sometime the full 
i will revert it and fix it